src/components/ValidatorView.js

```diff
@@ -37,6 +37,18 @@ function ValidatorViewController($scope, $rootScope, $log, $http, $window, $q, $
         className: 'alert-warning',
         content_tid: 'WARNING_V5_UNOFFICIAL'
       }]
+    },
+    'draft-06': {
+      service: validatorFactoryAJV('draft-06'),
+      name: 'draft-06 (latest)'
+    },
+    'Experimental': {
```

Comment:
The drafts, being officially titled as such, all start with lowercase, so I specifically went with uppercase both to have it stand apart from the other options on the menu and because I thought it looked better aesthetically (which is both subjective and not really that important, I realize). I will change it if you'd prefer to keep everything lowercase. 

This also brings up another issue: we currently have all the options listed chronologically. Would it be better to move v5-unnoficial to the bottom of the list, to really drive home the fact it shouldn't be treated like an official version? The official drafts all being grouped together would probably make the most sense.